Documentation/Contributing/development-environment.md

```diff
@@ -87,3 +87,12 @@ docker tag "local/ceph-$(go env GOARCH)" 'rook/ceph:master'
 
 4) Create a Rook cluster in minikube, or if the Rook cluster is already configured, apply the new
 operator image by restarting the operator.
+
+
+## Creating a dev cluster
+
+To accelerate the development process, users have the option to employ the script located
+at `tests/scripts/create-dev-cluster.sh`. This script is designed to rapidly set
```

Comment:
No args are needed, right? Perhaps just show a command block after this paragraph that makes it clear? I imagine in the future there could be args added as well for different options.

Reply:
> No args are needed, right? Perhaps just show a command block after this paragraph that makes it clear? I imagine in the future there could be args added as well for different options.

It does accept args to specify where the examples directory is but by default it will automatically pick the right directory. It also accept an optional arg to enable the rook operator. I didn't want to put the `usage` output since it could change in the future (new args i.e) so the docs will be come out of sync easily.